content/docs/operations/sample-app.md

```diff
@@ -0,0 +1,281 @@
+   
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:
+     name: httpbin
+     namespace: httpbin
+     labels:
+       example: httpbin-route
+   spec:
+     parentRefs:
+       - name: http
+         namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+     hostnames:
+       - "www.example.com"
+     rules:
+       - backendRefs:
+           - name: httpbin
+             port: 8000
+   EOF
+   ```
+
+   |Setting|Description|
+   |--|--|
+   |`spec.parentRefs`|The name and namespace of the gateway resource that serves the route. In this example, you use the HTTP gateway that you created earlier.  |
+   |`spec.hostnames`|A list of hostnames that the route is exposed on. In the example, the route is exposed on `www.example.com`. |
+   |`spec.rules.backendRefs`| The Kubernetes service that serves the incoming request. In this example, requests to `www.example.com` are forwarded to the httpbin app on port 9000. Note that you must create the HTTP route in the same namespace as the service that serves that route. To create the HTTP route resource in a different namespace, you must create a ReferenceGrant resource to allow the HTTP route to forward requests to a service in a different namespace. For more information, see the [Kubernetes API Gateway documentation](https://gateway-api.sigs.k8s.io/api-types/referencegrant/). |
+
+2. Verify that the HTTPRoute is applied successfully. 
+   
+   ```sh
+   kubectl get -n httpbin httproute/httpbin -o yaml
+   ```
+
+   Example output: Note the status of the HTTPRoute resource. Check for `Accepted` and `ResolvedRefs` messages. The `parentRef` refers to the Gateway that that HTTPRoute is exposed on.
+
+   ```yaml
+   status:
+     parents:
+     - conditions:
+       - lastTransitionTime: "2025-02-13T18:41:06Z"
+         message: ""
+         observedGeneration: 1
+         reason: Accepted
+         status: "True"
+         type: Accepted
+       - lastTransitionTime: "2025-02-13T18:41:06Z"
+         message: ""
+         observedGeneration: 1
+         reason: ResolvedRefs
+         status: "True"
+         type: ResolvedRefs
+       controllerName: kgateway.dev/kgateway
+       parentRef:
+         group: gateway.networking.k8s.io
+         kind: Gateway
+         name: http
+         namespace: kgateway-system
+   ```
+
+## Send a request {#send-request}
+
+Now that your httpbin app is running and you verified that the gateway resources are created, you can send a request to the app. The steps vary depending on your load balancer setup.
+
+{{< tabs items="Cloud Provider LoadBalancer,Port-forward for local testing" >}}
+{{% tab %}}
+1. Get the external address of the gateway and save it in an environment variable.
+   
+   ```sh
+   export INGRESS_GW_ADDRESS=$(kubectl get svc -n kgateway-system gloo-proxy-http -o=jsonpath="{.status.loadBalancer.ingress[0]['hostname','ip']}")
```

Comment:
```suggestion
   export INGRESS_GW_ADDRESS=$(kubectl get svc -n kgateway-system kgateway-proxy-http -o=jsonpath="{.status.loadBalancer.ingress[0]['hostname','ip']}")
```

Reply:
I think it's still called `gloo-proxy` unfortunately, at least for now: https://solo-io-corp.slack.com/archives/C08AAHU0RC0/p1739463892194579?thread_ts=1739303782.523689&cid=C08AAHU0RC0